pyccl/tracers.py

```diff
@@ -451,14 +540,15 @@ def __init__(self, cosmo, has_rsd, dndz, bias, mag_bias=None):
             a_s = 1./(1+z_b[::-1])
             t_a = (a_s, -growth_rate(cosmo, a_s))
             self.add_tracer(cosmo, kernel=kernel_d,
-                            transfer_a=t_a, der_bessel=2)
+                           transfer_a=t_a, der_bessel=2)         
```

Comment:
flake will complain about this change in indentation

Reply:
Done